packages/expo-modules-core/android/src/test/java/expo/modules/kotlin/ModuleHolderTest.kt

```diff
@@ -25,12 +27,19 @@ class ModuleHolderTest {
   }
 
   @Test
-  fun `should reject if method doesn't exist`() {
+  fun `should throw if method doesn't exist`() {
     val holder = ModuleHolder(EmptyModule())
     val promise = PromiseMock()
 
-    holder.call("not existing method", JavaOnlyArray(), promise)
-    Truth.assertThat(promise.state).isEqualTo(PromiseState.REJECTED)
-    Truth.assertThat(promise.rejectCode).isEqualTo("ERR_METHOD_NOT_FOUND")
+    assertThrows<MethodCallException>(
+      """
+      Cannot call `not existing method` from the `empty-module`.
```

Comment:
I guess this has more sense, because `from` indicated that the method is called by the `empty-module` and it should suggest that `empty-module` has not exposed `not existing method` via it's public API.
```suggestion
      Cannot call `not existing method` on the `empty-module`.
```